libraries/types/dag_block/src/dag_block.cpp

```diff
@@ -144,6 +162,7 @@ void DagBlock::streamRLP(dev::RLPStream &s, bool include_sig) const {
   if (include_sig) {
     s << sig_;
   }
+  s.appendVector(trx_estimations_);
```

Comment:
are we sure about that? I mean this should be calculated per each node and we should not append it to RLP. We malicious player could lie about the gas estimations

Reply:
If malicious peer would lie about the estimation it wouldn't pass verification on other nodes. Estimations should be the same for all nodes, because we are estimating transactions with dag block `proposal_period`, but not the latest block state(see [block_proposer.cpp:209](https://github.com/Taraxa-project/taraxa-node/pull/1672/files#diff-0240e8dd854ad98bf711fcd683da658fe26250cc6bf29d2c54b488b8a96dba60R209))

Reply:
I see, so another question does it need to be u256? because this has huge impact on DB and Dag size

Reply:
if it needs to be u256 do we need to store it as part of `SyncBlock`?

Reply:
Yeah, makes sense. I think we can make it `uint64_t` because I believe out block limit shouldn't be bigger then `uint64_t::max`